packages/appstore/src/modules/traders-hub/index.tsx

```diff
@@ -38,12 +38,11 @@ const TradersHub = observer(() => {
                 setScrolled(true);
             }, 200);
         }, 100);
+        // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
there's no cleanup for two timeouts and we've disabled the warning 🥺

Reply:
It is only applicable for non-EU client.